test/e2e/auth/service_accounts.go

```diff
@@ -153,6 +153,14 @@ var _ = SIGDescribe("ServiceAccounts", func() {
 		}
 	})
 
+	/*
+				Release: v1.9
+				Testname: Service Account Tokens Must AutoMount
```

Comment:
Why is the documented name different from the code description, i.e. "ServiceAccounts should mount an API token into pods". Seems like they should be consistent...

Reply:
@tallclair  I was just trying to generalize all 'service account token' related tests with that prefix. Please suggest better wording.
Service Account Tokens - must auto mount
Service Account Tokens - auto mount optionally ...

Reply:
Lots conformance tests replace spaces with dashes in the testname. Is the expected value / format / purpose of the testname documented anywhere?

Reply:
This test name here is the header in the conformance document for the test and format is a readable textual string.